features/step_definitions/logging.rb

```diff
@@ -53,41 +54,51 @@
 
   # project('openshift-logging')
   # Create operator group in `openshift-logging` namespace
-  opts[:operator_group_yaml] ||= "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/OCP-21311/operator-group.yaml"
-  @result = admin.cli_exec(:create, f: opts[:operator_group_yaml], n: logging_ns)
+  opts[:clo_operator_group_yaml] ||= "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/deploy_clo_via_olm/4.2/02_clo_og.yaml"
+  @result = admin.cli_exec(:create, f: opts[:clo_operator_group_yaml])
   raise "Error creating operatorgroup" unless @result[:success]
 
-  # Create the CatalogSourceConfig for the elasticsearch-operator in the namespace openshift-marketplace:
-
-  # create catalogsourceconfig for logging and elasticsearch
-  opts[:catsrc_logging_yaml] ||= "https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/openshift/ci-operator/build-image/cluster-logging-catalogsourceconfig.yaml"
-  opts[:catsrc_elasticsearch_yaml] ||= "https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/openshift/ci-operator/build-image/elasticsearch-catalogsourceconfig.yaml"
-
+  # Create operator group in `openshift-operators-redhat` namespace
+  opts[:eo_operator_group_yaml] ||= "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/eleasticsearch/deploy_via_olm/4.2/02_eo-og.yaml"
+  @result = admin.cli_exec(:create, f: opts[:eo_operator_group_yaml])
+  raise "Error creating operatorgroup" unless @result[:success]
 
-  @result = admin.cli_exec(:create, f: opts[:catsrc_logging_yaml])
```

Comment:
@QiaolingTang  it is not right to remove catsrc. we still neet to support 4.1. I am think if we can pull all resources in one yaml file.